src/js/chatWindow.js

```diff
@@ -1,8 +1,20 @@
-export const addMessageToChatHistory = (m) => {
+const guardOptions = (options) => {
+  if (options.messageType !== 'outgoing' && options.messageType !== 'incoming') {
+    return false;
+  }
+
+  return true;
+};
```

Comment:
I love how you included this guard to throw an error messages that aren't explicitly 'incoming' or 'outgoing'. Very nice!